kroxylicious-runtime/src/test/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandlerTest.java

```diff
@@ -95,6 +97,11 @@ private void writeRequest(short apiVersion, ApiMessage body) {
         inboundChannel.writeInbound(new DecodedRequestFrame<>(apiVersion, corrId, true, header, body));
     }
 
+    private <T extends ReferenceCounted> T track(T obj) {
```

Comment:
I think / would hope the netty leak detector extension would provide the same assurance 

Annotating this test with `@ExtendWith(NettyLeakDetectorExtension.class)` should do the job.